iis-wf/iis-wf-import/src/main/java/eu/dnetlib/iis/wf/importer/converter/OrganizationConverter.java

```diff
@@ -0,0 +1,97 @@
+package eu.dnetlib.iis.wf.importer.converter;
+
+import java.io.UnsupportedEncodingException;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.hadoop.hbase.client.Result;
+import org.apache.log4j.Logger;
+
+import com.google.common.base.Preconditions;
+
+import eu.dnetlib.data.proto.OafProtos.Oaf;
+import eu.dnetlib.data.proto.OafProtos.OafEntity;
+import eu.dnetlib.data.proto.OrganizationProtos.Organization.Metadata;
+import eu.dnetlib.iis.importer.schemas.Organization;
+
+/**
+ * Converter of {@link Oaf} object containing {@link eu.dnetlib.data.proto.OrganizationProtos.Organization}
+ * into {@link Organization}
+ * 
+ * 
+ * @author Łukasz Dumiszewski
+*/
+
+public class OrganizationConverter implements AvroConverter<Organization> {
+
+    
+    private static Logger log = Logger.getLogger(OrganizationConverter.class);
+    
+    
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Converts {@link Oaf} object containing {@link eu.dnetlib.data.proto.OrganizationProtos.Organization}
+     * into {@link Organization}
+     */
+    @Override
+    public Organization buildObject(Result source, Oaf resolvedOafObject) throws UnsupportedEncodingException {
+        
+        
+        Preconditions.checkNotNull(resolvedOafObject);
+        Preconditions.checkNotNull(source);
```

Comment:
We shouldn't care about source in this context. If `resolvedOafObject` is not null we are fine.